.github/workflows/firestore.yml

```diff
@@ -202,7 +202,8 @@ jobs:
       run: |
         scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb ${{ matrix.podspec }} \
             --platforms=ios \
-            --allow-warnings
+            --allow-warnings \
+            --no-analyze
```

Comment:
Please open an issue to reenable analyze

Reply:
Done.